src/main/app/forms/models/address.ts

```diff
@@ -71,7 +72,7 @@ export class Address implements CompletableTask {
   @IsNotBlank({ message: ValidationErrors.POSTCODE_REQUIRED, groups: ['claimant', 'defendant', 'response'] })
   @IsCountrySupported(Country.all(), { message: ValidationErrors.CLAIMANT_COUNTRY_NOT_SUPPORTED, groups: ['claimant'] })
   @IsCountrySupported(Country.defendantCountries(), { message: ValidationErrors.DEFENDANT_COUNTRY_NOT_SUPPORTED, groups: ['defendant'] })
-  @MaxLength(ValidationConstants.POSTCODE_MAX_LENGTH, {
```

Comment:
have you removed this const from `ValidationConstants`?

Reply:
done